packages/test/test-end-to-end-tests/src/test/stashedOps.spec.ts

```diff
@@ -106,6 +113,35 @@ const getPendingOps = async (args: ITestObjectProvider, send: boolean, cb: MapCa
     return pendingState;
 };
 
+const getPendingCellOps = async (args: ITestObjectProvider, send: boolean, cb: CellCallback) => {
```

Comment:
I wouldn't duplicate `getPendingOps()`, you can use it with SharedCell by using `getSharedObject()` on the DataStore like in the SharedString tests in this file.

It would make sense to remove the MapCallback type and make getPendingOps more generic, but you don't have to do that in this PR.

Reply:
Agree. I was trying to do that but got some issues with typescript. I came with this approach just because it was the least time consuming.

Reply:
I don't think adding this function is good since it does essentially the same thing as getPendingOps(), so any change to one would need to be made twice.

You can just use getPendingOps() the same way as the SharedString tests in this file:
``` ts
        const pendingOps = await getPendingOps(provider, false, async (c, d, map) => {
            const s = await d.getSharedObject<SharedString>(stringId);
            s.insertText(s.getLength(), " world!");
        });
```
A further step would be to do the same thing in all the SharedMap tests and remove the map callback argument, but that's not directly related to the issue this PR addresses so it doesn't need to be done in this PR.